### PR TITLE
feat: create calculation page

### PR DESCRIPTION
**Pull request by hannest03**

### CODE REVIEW

File: app/src/app/app.module.ts [hunk 1]
```diff
@@ -32,19 +38,25 @@ import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
     CalculationListComponent,
     CalculationCardComponent,
     SpecificCalculationListComponent,
-	CalculationDiagramComponent
+	  CalculationDiagramComponent,
```
Comment (Nis5l) [inline app/src/app/app.module.ts:44]: Wrong indentation, please fix....

File: app/src/app/create-calculation/create-calculation.component.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<div class="container">
+    <form [formGroup]="createForm" (ngSubmit)="create()" fxLayout="column">
+        <mat-form-field appearance="fill">
+            <mat-label>Name</mat-label>
+            <input matInput placeholder="Name" formControlName="name"><mat-error
```
Comment (Nis5l) [inline app/src/app/create-calculation/create-calculation.component.html:5]: Please use newline on mat-error

File: app/src/app/create-calculation/create-calculation.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (Nis5l) [inline app/src/app/create-calculation/create-calculation.component.spec.ts:1]: Remove useless spec file

**Review by Nis5l — CHANGES_REQUESTED** [review]

File: app/src/app/app.module.ts [hunk 4]
```diff
@@ -47,11 +51,16 @@ import { CreateCalculationComponent } from './create-calculation/create-calculat
       echarts: () => import('echarts')
     }),
     BrowserAnimationsModule,
-    MatIconModule,
     MatButtonModule,
     MatInputModule,
     MatFormFieldModule,
     ReactiveFormsModule,
+	  FlexLayoutModule,
```
Comment (Nis5l) [inline app/src/app/app.module.ts:58]: Fix indentation please...

**Review by Nis5l — CHANGES_REQUESTED** [review]

File: app/src/app/create-calculation/create-calculation.component.html (hunk 2 above)
Comment (hannest03) [reply app/src/app/create-calculation/create-calculation.component.html]: fixed sorry will never happen again D:

Comment (Nis5l) [reply app/src/app/create-calculation/create-calculation.component.html]: I sure hope so, otherwise this might be your last pull request.

**Review by Nis5l — APPROVED** [review]

**Pull request merged**